ggplot_base_r_graph_analysis.Rmd

```diff
@@ -0,0 +1,115 @@
+---
+title: "community contribution"
+author: "zezhong fan"
+date: "10/29/2021"
+output: html_document
+---
+```{r}
+library(ggplot2)
+library(dplyr)
+library(tidyverse)
+library(tidyr)
+```
+***Bar Chart***
```

Comment:
Suggestion: to create subheadings you can change the '***' to '##'
Remember  just two hashtags, (don't mark subheading with a single #)